pod.go

```diff
@@ -998,6 +998,10 @@ func (p *Pod) checkContainersState(state stateString) error {
 	return nil
 }
 
+// Drive names are dependent on the order in which they are added to the hypervisor options.
+// driveIndex is an incrementing counter used to track the index at which current drive is added to the hypervisor.
+var driveIndex = int(0)
```

Comment:
We can do better than this, by adding this index to the hypervisor structure maybe. There is clearly no need for a global variable here. And more than that, keep in mind that we don't need to store this data because this will only be used by CreatePod(). Maybe this will be interesting to store in case we consider hotplugging some new block devices when the VM is already started.